core/src/main/java/co/cask/wrangler/parser/TextDirectives.java

```diff
@@ -1039,8 +1009,20 @@ public TextDirectives(List<String> directives) {
         }
         break;
 
+        //reverse <column>
+        case "reverse": {
+          String col = getNextToken(tokenizer, command, "col", lineno);
+          steps.add(new Reverse(lineno, directive, col));
+        }
+        break;
 
-
+        //rowhash <column>
+        case "rowhash": {
+          String col = getNextToken(tokenizer, command, "col", lineno);
+          String codec = getNextToken(tokenizer, command, "codec", lineno);
+          steps.add(new RowHash(lineno, directive, col, codec));
```

Comment:
if you hash with line number if he contents are the the same in two different lines, it will generate two different hashes. is that what you want? in which case it is more of a unique id generator and not a hash.